tests/Message/AbstractRequestTest.php

```diff
@@ -420,6 +420,21 @@ public function testPrices()
         $this->assertSame($prices, $this->request->getPrices());
     }
 
+    public function testDefaultParameterOnCreate() : void
+    {
+      $request = Mocker::mock('\Omnipay\Vindicia\Message\AbstractRequest')->makePartial()->shouldAllowMockingProtectedMethods();
+      $request->initialize();
+      $this->assertSame(AbstractRequest::DEFAULT_TAX_CLASSIFICATION, $request->getTaxClassification());
+    }
+
+    public function testDefaultParameterOnUpdate() : void
```

Comment:
Same issue here mentioned in https://github.com/vimeo/omnipay-vindicia/pull/63/files#r352650949